File: 2.0/python/me/getCreditUsage.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
response = client.me.getCreditUsage(
    # tim in unix milliseconds
    intervalBegin=2145905600000,
    intervalEnd=2145945600000
).result()

print(response)

File: 2.0/php/me/getCreditUsage.php
<?php

class ApiClientSample {

    public static function main() {
        $client = \CallFire\Api\DocumentedClient::createClient("account_login", "account_password");
        $request = $client->getCreditUsage();
        $request->getOperationConfig()->setQueryParameters(array("intervalBegin" => 2145905600000,
                                                                 "intervalEnd" => 2145945600000));
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();
